sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_param.json

```diff
@@ -131,31 +131,43 @@
     "spark_runtime_version" : {
         "value": ""
     },
+    "offlineStoreStorageAccountOption": {
+        "value": "none"
```

Comment:
is it the default value? should it be null or "new"?

Reply:
this is default value